app/src/main/java/com/example/hungrywolfs/model/AppViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.hungrywolfs.model
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.hungrywolfs.network.MealApi
+import com.example.hungrywolfs.network.Category
+import com.example.hungrywolfs.network.Meal
+import kotlinx.coroutines.launch
+
+class AppViewModel : ViewModel() {
+    private val _foodInfo = MutableLiveData<Category>()
+    val info: LiveData<Category> = _foodInfo
+
+    private val _mealInfo = MutableLiveData<Meal>()
+    val mealInfo: LiveData<Meal> = _mealInfo
+
+    init {
+        getCategories()
+    }
+
+    private fun getCategories() {
+        viewModelScope.launch {
+            try {
+                _foodInfo.value = MealApi.retrofit_service.getMealCategories()
```

Comment:
We don't really use underscore with lowercase when naming fields - this should be `MealApi.retrofitService` or full uppercase `MealApi.RETROFIT_SERVICE`. I'd take the first one cuz most of the time just simple constants are named with uppercase, like

`const val SCREEN_DELAY = 3000L`